packages/rainbowkit/src/hooks/useEnsChain.ts

```diff
@@ -0,0 +1,13 @@
+import { chain } from 'wagmi';
+import { useRainbowKitChains } from './../components/RainbowKitProvider/RainbowKitChainContext';
+
+export function useEnsChain() {
+  const rainbowKitChains = useRainbowKitChains();
+
+  const chainId = chain.mainnet.id;
+  const enabled = rainbowKitChains.some(
+    rainbowKitChain => rainbowKitChain.id === chainId
+  );
+
+  return { chainId, enabled };
+}
```

Comment:
nit: since the `chainId` is hardcoded to mainnet, should this hook be named `useMainnetEnsChain`?

Reply:
nice, yeah that makes more sense to me now <3